react/features/shared-video/constants.js

```diff
@@ -22,6 +22,13 @@ export const YOUTUBE_PLAYER_PARTICIPANT_NAME = 'YouTube';
  */
 export const SHARED_VIDEO = 'shared-video';
 
+/**
+ * Shared video type.
+ *
+ * @type {string}
+ */
+export const VIDEO = 'video';
```

Comment:
Do we need this? Can't we use `MEDIA_TYPE.VIDEO` ?

Reply:
done